pom.xml

```diff
@@ -31,6 +31,10 @@
             <id>jaudiotagger-repository</id>
             <url>https://dl.bintray.com/ijabz/maven</url>
         </repository>
+        <repository>
+            <id>jitpack.io</id>
+            <url>https://jitpack.io</url>
+        </repository>
```

Comment:
From eigenholser in #24 : Which dependency does this support.

Reply:
`<groupId>com.github.goxr3plus</groupId>
            <artifactId>jaudiotagger</artifactId>
            <version>2.2.7</version>`
jitpack seems to support jaudiotagger. Documentation is available here: jitpack.io